src/editor/standardeditor.js

```diff
@@ -8,16 +8,13 @@
  */
 
 import Editor from './editor';
-import EditingKeystrokeHandler from '../editingkeystrokehandler';
-import EditingController from '@ckeditor/ckeditor5-engine/src/controller/editingcontroller';
 import isFunction from '@ckeditor/ckeditor5-utils/src/lib/lodash/isFunction';
 
 import getDataFromElement from '@ckeditor/ckeditor5-utils/src/dom/getdatafromelement';
 import setDataInElement from '@ckeditor/ckeditor5-utils/src/dom/setdatainelement';
 
 /**
- * Class representing a typical browser-based editor. It handles a single source element and
- * uses {@link module:engine/controller/editingcontroller~EditingController}.
+ * Class provides API methods for getting and setting data.
```

Comment:
The standard editor class, extending base editor by adding methods needed in the typical editor implementations: editor with a single editable area created based on the DOM element. It provides base API to manage data and to integrate the editor with the form when it is initialized on the textarea element.